src/components/MenuItem/__spec__/menuItem.spec.tsx

```diff
@@ -1,14 +1,17 @@
 import React from "react";
 import { render, screen, cleanup } from "@testing-library/react";
 import MenuItem from "../MenuItem";
+import { BrowserRouter } from "react-router-dom";
 
 afterEach(() => {
   cleanup();
 });
 
 test("should render menuItem", () => {
-  render(<MenuItem item="string" />);
-  const menuItemElement = screen.getByTestId("menu-item");
-
-  expect(menuItemElement).toBeInTheDocument();
+  <BrowserRouter>
+    render(
+    <MenuItem title={""} path={""} />
+    ); const menuItemElement = screen.getByTestId("menu-item");
+    expect(menuItemElement).toBeInTheDocument();
+  </BrowserRouter>;
```

Comment:
that is wrong code. you have to put BrowserRouter inside of the render method. and js code outside of BrowserRouter Wrapper. this test doesnt work! same for others tests

Reply:
But this test is working!